accepted/future-releases/contravariant-superinterface-2018/implementation-plan.md

```diff
@@ -0,0 +1,30 @@
+# Implementation Plan for the Q1 2019 Superinterface Contravariance Error.
+
+Relevant documents:
+ - [Tracking issue](https://github.com/dart-lang/language/issues/113)
+ - [Feature specification](https://github.com/dart-lang/language/blob/master/accepted/future-releases/contravariant-superinterface-2018/feature-specification.md)
+
+
+## Implementation and Release plan
```

Comment:
I think we should use an experiment flag, even though breakage chances are low.  It makes sure we roll out consistently.  Maybe "covariant-only-superinterfaces"?

Reply:
+1 to using that flag name

Reply:
Added flag to plan.